core-web/libs/block-editor/src/lib/extensions/ai-content-actions/ai-content-actions.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Observable } from 'rxjs';
+
+import { Component, EventEmitter, Output, OnInit } from '@angular/core';
+
+import { AiContentService } from '../../shared/services/ai-content/ai-content.service';
+
+interface ActionOption {
+    label: string;
+    icon: string;
+    selectedOption: boolean;
+    callback: () => void;
+}
+
+export enum ACTIONS {
```

Comment:
Too generic... pleace use a more specific name to avoid collisions with other libs and apps.